versions/latest/modules/en/nav.adoc

```diff
@@ -8,6 +8,8 @@
 
 * xref:virtualclusterpolicy.adoc[VirtualClusterPolicy]
 
+* xref:k3k.adoc[K3k: Kubernetes in Kubernetes]
```

Comment:
I think needs to be placed earlier? Many of the files above it are already making references to K3k, but the first few lines of this file (k3k.adoc) seem to be introducing what it is for the first time.